examples/example-vite-grpc-playground/src/components/query-results.tsx

```
@@ -0,0 +1,107 @@
+import { Alert } from "./ui/alert";
+import { Button } from "./ui/button";
+
+interface QueryResultsProps {
+    response: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)